gcn_email_service/__init__.py

```diff
@@ -100,7 +99,13 @@ def recieve_alerts(consumer):
 @on_exception(expo, RateLimitException)
 @limits(calls=MAX_SENDS, period=1)
 def send_raw_ses_message_to_recipient(client, message, recipient):
-    BODY_TEXT = str(email.message_from_bytes(message.value()))
+    BODY_TEXT = ""
+    ATTACHMENT = False
+    if '.text.' in message.topic():
+        BODY_TEXT = str(email.message_from_bytes(message.value()))
```

Comment:
What does this line do? Why is `body_text = message.value()` or `body_text = message.value().decode()` not sufficient?

Reply:
Trying out the other options now. Originally there were some issues because of the other formats, but since they are attachments it no longer applies